src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,46 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * <h1> Event </h1>
+ * Encapsulates a task that has a description and a date and time that it occurs on.
+ *
+ * @author Clifford
+ */
+public class Event extends Task {
+    protected static final String TASK_SYMBOL = "[E]";
+    protected static final DateTimeFormatter PARSE_FORMAT = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+    protected static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy, hh:mm a");
+    protected LocalDateTime dateTime;
+
```

Comment:
Is this an extra line? (I'm not too sure...)

Reply:
Indeed it is. I have made the necessary corrections